intermediate/challenge-1/src/main/java/com/byond/shifts/service/schedule/bus/BusSchedulerPaginationRepository.java

```diff
@@ -0,0 +1,14 @@
+package com.byond.shifts.service.schedule.bus;
+
+import com.byond.shifts.service.schedule.Schedule;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.repository.PagingAndSortingRepository;
+import org.springframework.stereotype.Repository;
+
+import java.time.DayOfWeek;
+
+@Repository
+public interface BusSchedulerPaginationRepository extends PagingAndSortingRepository<Schedule, Integer> {
```

Comment:
I'm finding it difficult to follow the naming of `BusScheduler`: maybe if it was just  a `BusSchedule`, then we know we are looking at 'existing schedules for buses'